src/mds/MDCache.cc

```diff
@@ -11648,6 +11654,7 @@ void MDCache::merge_dir(CInode *diri, frag_t frag)
 
 void MDCache::fragment_freeze_dirs(const std::vector<CDir*>& dirs)
 {
+  ceph_assert(kill_dirfrag_at != 1);
```

Comment:
Can these killpoints be defined as an enumerated value rather than hard-coding them this way? I presume killpoints reflect various stages that a dirfrag undergoes fragmentation. Having them nicely names would be helpful (during reviews).